codecs/avif/Makefile

```diff
@@ -68,24 +74,58 @@ $(OUT_NODE_DEC_JS) $(OUT_DEC_JS): $(OUT_DEC_CPP) $(CODEC_DIR)/CMakeLists.txt $(L
 			-DCONFIG_MULTITHREAD=0 \
 		" \
 		ENVIRONMENT=$(ENVIRONMENT) \
-		LIBAVIF_FLAGS="-DAVIF_CODEC_AOM_ENCODE=0"
+		LIBAVIF_FLAGS="-DAVIF_CODEC_AOM_ENCODE=0 -DAVIF_LOCAL_LIBSHARPYUV=ON"
```

Comment:
I believe we can omit -DAVIF_LOCAL_LIBSHARPYUV=ON in the decoding-only build. Sharp YUV is only used in the RGB-to-YUV conversion, which is only used when preparing the YUV input to the AVIF encoder.